earthly-entrypoint.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh
+
+set -e
+
+# Apply global configuration
+if [ -n "$GLOBAL_CONFIG" ]; then
+  earthly config global "$GLOBAL_CONFIG"
+fi
+
+# Apply git configuration
+if [ -n "$GIT_CONFIG" ]; then
+  earthly config git "$GIT_CONFIG"
+fi
```

Comment:
I've seen other images allow to mount a config file instead. It might be cleaner and more future-proof as just a file maybe?

Reply:
I went back-and-forth here. Nothing is stopping you from mounting that as-is; but, as an example within our own testing, the mount option is unavaliable, and this felt nicer than a bunch of `RUN earthly config` lines.

I'll invoke `earthly` with `--config /some/well/known/path`, and document that you can mount it too. These options will stay for when mounting isn't an option.